suite/tests/CMakeLists.txt

```diff
@@ -2121,6 +2121,16 @@ if (CLIENT_INTERFACE)
       "" "")
   endif (NOT ARM)
 
+  add_library(client.blackbox.privdll SHARED client-interface/blackbox.privdll.c)
+  configure_DynamoRIO_client(client.blackbox.privdll)
+  add_library(client.blackbox.dll SHARED client-interface/blackbox.dll.c)
+  configure_DynamoRIO_client(client.blackbox.dll)
+  target_link_libraries(client.blackbox.dll client.blackbox.privdll)
+  set_target_properties(client.blackbox.dll PROPERTIES SKIP_BUILD_RPATH OFF)
```

Comment:
Why is this necessary, and in particular for just one of the clients?  Please explain in a comment as this seems unusual.

Reply:
Done.